# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/xjdic/errno.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -pruN tmp_orig/exjdxgen.c tmp/exjdxgen.c
--- tmp_orig/exjdxgen.c	1998-05-24 17:43:37.000000000 +0200
+++ tmp/exjdxgen.c	2004-09-24 21:36:17.785523480 +0200
@@ -25,6 +25,7 @@
 /*#include <stdlib.h>*/
 #include <ctype.h>
 #include <string.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define TRUE 1
diff -pruN tmp_orig/makekanjstroke.c tmp/makekanjstroke.c
--- tmp_orig/makekanjstroke.c	1996-01-20 06:38:30.000000000 +0100
+++ tmp/makekanjstroke.c	2004-09-24 21:36:20.687082376 +0200
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include <errno.h>
 
 FILE *fi,*fo,*fopen();
 unsigned char instr[1000],*ptr;
diff -pruN tmp_orig/xjdclient.c tmp/xjdclient.c
--- tmp_orig/xjdclient.c	1998-08-31 12:56:56.000000000 +0200
+++ tmp/xjdclient.c	2004-09-24 21:36:24.014576520 +0200
@@ -36,6 +36,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <signal.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define CVERBOSE 0
diff -pruN tmp_orig/xjdsa.c tmp/xjdsa.c
--- tmp_orig/xjdsa.c	1998-08-30 02:31:20.000000000 +0200
+++ tmp/xjdsa.c	2004-09-24 21:36:36.367698560 +0200
@@ -28,6 +28,7 @@
 #include <sys/stat.h>
 #include <stdlib.h>
 #include <signal.h>
+#include <errno.h>
 #include "xjdic.h"
 
 unsigned char Dnamet[10][100],XJDXnamet[10][100];
diff -pruN tmp_orig/xjdserver.c tmp/xjdserver.c
--- tmp_orig/xjdserver.c	2003-05-28 07:58:02.000000000 +0200
+++ tmp/xjdserver.c	2004-09-24 21:36:43.778571936 +0200
@@ -28,6 +28,7 @@
 #include <netdb.h>
 #include <stdlib.h>
 #include <signal.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define SVERBOSE 0
diff -pruN tmp_orig/xjdxgen.c tmp/xjdxgen.c
--- tmp_orig/xjdxgen.c	2003-05-28 07:58:29.000000000 +0200
+++ tmp/xjdxgen.c	2004-09-24 21:36:46.786114720 +0200
@@ -26,6 +26,7 @@
 /*#include <stdlib.h>*/
 #include <ctype.h>
 #include <string.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define TRUE 1
